Documentation/ColumnsConfig/Type/Flex.rst

```diff
@@ -388,11 +388,11 @@ format. The structure is as follows:
 
 .. _columns-flex-data-format-array-sheet:
 
-<sheet>
+<sheets>
 ^^^^^^^
```

Comment:
The "underline" of the headline has to be as long as the headline itself. In this case the headline was increased by one character, so underline has to be increased also by one.